### PR TITLE
CNV-38476: Update runbooks VirtualMachineCRCErrors

### PR DESCRIPTION
**Pull request by sbeskin-redhat**
Resolves: https://issues.redhat.com/browse/CNV-38476

OCP 4.14+

Preview: https://73108--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/monitoring/virt-runbooks#virt-runbook-VMStorageClassWarning

### CODE REVIEW

**Comment by sbeskin-redhat** [comment]
@sradco  Please review this PR.

**Comment by sbeskin-redhat** [comment]
Approved by @Ahmad-Hafe

File: modules/virt-runbook-vmstorageclasswarning.adoc [hunk 1]
```diff
@@ -54,29 +54,10 @@ $ oc get pv <pv_name> -o=jsonpath='{.spec.storageClassName}'
...
-  mounter: rbd
-  mapOptions: "krbd:rxbounce"
-provisioner: openshift-storage.rbd.csi.ceph.com
-# ...
-----
-
-The `krbd:rxbounce` option creates a bounce buffer to receive data. The default
-behavior is for the destination buffer to receive data directly. A bounce buffer
-is required if the stability of the destination buffer cannot be guaranteed.
+Create a default OpenShift Virtualization storage class with the"krbd:rxbounce" map option. See  link:https://access.redhat.com/articles/6978371[Optimizing ODF PersistentVolumes for Windows VMs] for details.
```
Comment (sradco) [inline modules/virt-runbook-vmstorageclasswarning.adoc:60]: @sbeskin-redhat The link to "Optimizing ODF PersistentVolumes for Windows VMs" is referenced here twice..

Comment (sbeskin-redhat) [reply modules/virt-runbook-vmstorageclasswarning.adoc:60]: @sradco  Fixed, thank you

**Comment by mburke5678** [comment]
@sbeskin-redhat This might be a dumb question. Is this statement still relevant here: _// Do not edit this module. It is generated with a script._? Is this file essentially a placeholder until it is generated by the script? I'm a bit confused.

File: modules/virt-runbook-vmstorageclasswarning.adoc [hunk 2]
```diff
@@ -5,26 +5,26 @@
 // * virt/monitoring/virt-runbooks.adoc
```
Comment (apinnick) [inline modules/virt-runbook-vmstorageclasswarning.adoc:5]: Did you read the comments? "Do not edit this module. It is generated with a script." Did anyone tell you about the runbook conversion script?

Comment (mburke5678) [reply modules/virt-runbook-vmstorageclasswarning.adoc:5]: Was that ^^ for @sbeskin-redhat ?

Comment (apinnick) [reply modules/virt-runbook-vmstorageclasswarning.adoc:5]: Yes

Comment (sbeskin-redhat) [reply modules/virt-runbook-vmstorageclasswarning.adoc:5]: @apinnick 
Yes, i know about the script. However, my instructions from @ctomasko were as follows:
"You do not have to sync any scripts to update this runbook. You can open a PR and make this change. We need to update the downstream runbook to reflect the change that was made upstream."

**Pull request merged**